src/main/java/io/logz/benchmarks/elasticsearch/elasticsearch/ElasticsearchController.java

```diff
@@ -276,4 +291,28 @@ private JestClient initializeJestClient(ElasticsearchConfiguration esConfig) {
 
         return tempFilesContentList;
     }
+
+    private ArrayList<String> getExternalDirectoryContent(String path) {
+        ArrayList<String> tempFilesContentList = new ArrayList<>();
+
+        File directory = new File(path);
+        if (!directory.isDirectory()) {
+            throw new RuntimeException("Did not find any files under "+ path +"!");
+        }
+
+        File[] documents = directory.listFiles();
+
+        if (documents == null || documents.length == 0) {
+            throw new RuntimeException("Did not find any files under "+ path +"!");
+        }
+
+        for (File doc : documents) {
+            try {
+                tempFilesContentList.add(String.join("", Files.readLines(doc, StandardCharsets.UTF_8)));
+            } catch (IOException e) {
+                logger.info("Could not read file {}", doc.getAbsolutePath());
+            }
+        }
+        return tempFilesContentList;
```

Comment:
Need another RuntimeException here if after iterating on all files this is still empty (in case IOException while reading)